hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -722,6 +722,9 @@ msgstr ""
 "Es gab einen Fehler bei der Bearbeitung Deine Anfrage. Bitte versuche es "
 "noch einmal."
 
+#~ msgid "Please enter a value in range 2 - 200."
+#~ msgstr "Bitte gib einen Wert von 2 bis 200 ein."
+
```

Comment:
this can be deleted too.

Reply:
Leaving this for a later job. To do a cleanup.